scripts/build.js

```diff
@@ -2,23 +2,28 @@
 const { spawn } = require("child_process")
 
 const axios = require("axios")
-const btoa = require("btoa")
 
-// login credentials
-const { PERSONAL_ACCESS_TOKEN, USERNAME } = process.env
-if (!PERSONAL_ACCESS_TOKEN || !USERNAME) {
+// Login credentials
+const {
+  CYPRESS_COOKIE_NAME,
+  CYPRESS_COOKIE_VALUE,
+  CYPRESS_BACKEND_URL,
+} = process.env
+if (!CYPRESS_COOKIE_NAME || !CYPRESS_COOKIE_VALUE) {
   throw new Error(
-    `E2E tests require env vars PERSONAL_ACCESS_TOKEN (${PERSONAL_ACCESS_TOKEN}) and USERNAME (${USERNAME}) to be set.`
+    `E2E tests require env vars CYPRESS_COOKIE_NAME (${CYPRESS_COOKIE_NAME}) and CYPRESS_COOKIE_VALUE (${CYPRESS_COOKIE_VALUE}) to be set.`
```

Comment:
Nit: lets not throw/log the actual content of an env var, better practice here would just be to mention that `E2E tests require env vars CYPRESS_COOKIE_NAME and CYPRESS_COOKIE_VALUE to be set`

Reply:
Fixed in c9c68b837c0485d6f7d4859b9eaac88a3d2c6c14!